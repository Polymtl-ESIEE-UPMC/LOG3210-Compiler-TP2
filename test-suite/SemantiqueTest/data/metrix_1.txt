Declaration
num a;
num b;
EndDeclaration

a = 1;
b = 2;
a = b + 1;
