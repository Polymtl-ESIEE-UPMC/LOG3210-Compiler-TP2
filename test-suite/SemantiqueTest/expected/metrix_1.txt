{VAR:2, WHILE:0, IF:0, OP:1}